lib/matplotlib/backends/backend_pgf.py

```diff
@@ -67,7 +67,7 @@ def _get_preamble():
                 # 1) Forward slashes also work on Windows, so don't mess with
                 # backslashes.  2) The dirname needs to include a separator.
                 path = pathlib.Path(fm.findfont(family))
-                preamble.append(r"\%s{%s}[Path=\detokenize{%s/}]" % (
+                preamble.append(r"\{}{{{}}}[Path=\detokenize{{{}/}}]".format(
```

Comment:
most of the f-strings in this file are also harder to read than their ~%` formatted counterparts due to the prevalence of braces in the interpolated strings